docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -233,408 +122,121 @@ is used.
-Deleting orphan files from time to time is recommended to keep size of table's data directory under control.
-
-``remove_orphan_files`` can be run as follows:
-
-.. code-block:: sql
-
-  ALTER TABLE test_table EXECUTE remove_orphan_files(retention_threshold => '7d')
-
-The value for ``retention_threshold`` must be higher than or equal to ``iceberg.remove_orphan_files.min-retention`` in the catalog
-otherwise the procedure will fail with similar message:
-``Retention specified (1.00d) is shorter than the minimum retention configured in the system (7.00d)``.
-The default value for this property is ``7d``.
-
-.. _drop-extended-stats:
-
-drop_extended_stats
-~~~~~~~~~~~~~~~~~~~
-
-The ``drop_extended_stats`` command removes all extended statistics information from
-the table.
-
-``drop_extended_stats`` can be run as follows:
-
-.. code-block:: sql
-
-  ALTER TABLE test_table EXECUTE drop_extended_stats
-
-.. _iceberg-alter-table-set-properties:
-
-ALTER TABLE SET PROPERTIES
-~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The connector supports modifying the properties on existing tables using
-:ref:`ALTER TABLE SET PROPERTIES <alter-table-set-properties>`.
+.. _iceberg-hive-catalog:
 
-The following table properties can be updated after a table is created:
+Hive metastore catalog
+^^^^^^^^^^^^^^^^^^^^^^
 
-* ``format``
-* ``format_version``
-* ``partitioning``
-* ``sorted_by``
+The Hive metastore catalog is the default implementation.
+When using it, the Iceberg connector supports the same metastore
+configuration properties as the Hive connector. At a minimum,
+``hive.metastore.uri`` must be configured, see
+:ref:`Thrift metastore configuration<hive-thrift-metastore>`.
 
-For example, to update a table from v1 of the Iceberg specification to v2:
+.. code-block:: text
 
-.. code-block:: sql
+    connector.name=iceberg
+    hive.metastore.uri=thrift://localhost:9083
 
-    ALTER TABLE table_name SET PROPERTIES format_version = 2;
+.. _iceberg-glue-catalog:
 
-Or to set the column ``my_new_partition_column`` as a partition column on a table:
+Glue catalog
+^^^^^^^^^^^^
 
-.. code-block:: sql
+When using the Glue catalog, the Iceberg connector supports the same
+configuration properties as the Hive connector's Glue setup. See
+:ref:`AWS Glue metastore configuration<hive-glue-metastore>`.
 
-    ALTER TABLE table_name SET PROPERTIES partitioning = ARRAY[<existing partition columns>, 'my_new_partition_column'];
+.. code-block:: text
 
-The current values of a table's properties can be shown using :doc:`SHOW CREATE TABLE </sql/show-create-table>`.
+    connector.name=iceberg
+    iceberg.catalog.type=glue
 
-COMMENT
-~~~~~~~
+.. _iceberg-rest-catalog:
 
-The Iceberg connector supports setting comments on the following objects:
+REST catalog
+^^^^^^^^^^^^^^
 
-- tables
-- views
-- table columns
+In order to use the Iceberg REST catalog, ensure to configure the catalog type with
```

Comment:
This is inconsistent with the other two before.. we should rejig

Reply:
Also .. we recently added some info about limitation for JDBC and REST.. have those gone lost? Or did I just not see it or misremember

Reply:
This is existing content, which we promised not to change. This is only a reorganization of what currently exists. If there are changes to content they have to be done in a separate PR.